src/components/CreateHabitForm/index.tsx

```diff
@@ -25,9 +25,18 @@ const ME = gql`
   }
 `;
 
+const CREATE_GROUP_HABIT = gql`
+  mutation CREATE_GROUP_HABIT($group_id: String, $input: HabitInput) {
+    createGroupHabit(group_id: $group_id, input: $input) {
+      habit_name
+    }
+  }
+`;
+
 const CreateHabitForm = ({ refetch }) => {
   const [pressed, setPressed] = useState(false);
   const [createHabit, { data, loading, error }] = useMutation(CREATE_HABIT);
+  const [createGroupHabit, { data: gData, loading: gLoading, error: gError }] = useMutation(CREATE_GROUP_HABIT);
```

Comment:
Be a little more verbose here. Longer variable names are fine. Something like `createGroupData` and `createGroupLoading`. So it's a bit more clear

Reply:
Fixed Here: https://github.com/osu-cascades/habit-loop/pull/74/commits/04aa09a4e845f01f420dc998712cbdf08962104e